### ISSUE TITLE
[TCotD2] Future interest in 2.1?

### ISSUE THREAD

**Issue by Tormentor667** [Realm667/Re-Releases#290]
@Blue-Shadow @nashmuhandes @Ozymandias81 I just read over the two long replies over at https://www.moddb.com/mods/the-city-of-the-damned-2-apocalypse/downloads/the-city-of-the-damned-2-apocalypse-v20/#8652678 with a lot of valuable feedback and I wonder if you are interested in spending some more work into TCotD2? Some of the thoughts are pretty good and I think we could polish the mod furthermore.

**Comment by Blue-Shadow** [comment]
Though not immediately, I, myself, will be leaving the community, so I won't be contributing further to this or any other project. I may return one day, or I may not. Time will tell.

**Comment by Tormentor667** [comment]
Sad to hear… specific reasons?

**Comment by Blue-Shadow** [comment]
> Sad to hear… specific reasons?

In short and without going into details, for one reason or another, I've lost the drive and passion for doing whatever things I do around the community.

**Comment by nashmuhandes** [comment]
After thinking a bit about this (and reading some of the comments on both the ZD forum and ModDB)... I _suppoooooooooooooose_ we could do one final patch. Strictly fixes only (says the person who has trouble resisting feature creep X)).

We'll take a very minimal-editing approach to this.

How am I supposed to update the project? It got moved into a "completed" folder. Do I just work from there? @Tormentor667

**Comment by Tormentor667** [comment]
Yes you can directly work from that folder :) Anything I can work and help with?

**Comment by nashmuhandes** [comment]
Latest playthrough issue dump

- remove collision from bones/other decorative crap
- combat balance is still crap especially in single player
- cistern ticking script somehow not consistently heard in coop
- coop player-grouping script not working for the spider room
